js/ui/viewSelector.js

```diff
@@ -445,6 +449,33 @@ const ViewsClone = new Lang.Class({
                                    transition: 'easeOutQuad' });
             }
         }));
+
+        this._settings.connect('changed::' + KEY_TEXT_SCALING_FACTOR, Lang.bind(this, function() {
+            /*
+             * Add it in an idle to give Clutter some time to
+             * recalculate whatetever it needs to. Simply queueing
+             * redraw right after the setting changes doesn't
+             * work.
+             */
+            GLib.idle_add(GLib.PRIORITY_DEFAULT, Lang.bind(this, function() {
+                let overviewVisible = Main.layoutManager.overviewGroup.visible;
+                let saturationEnabled = this._saturation.enabled;
+
+                this._saturation.enabled = false;
```

Comment:
Are you disabling the saturation for the same reason in the comment below? If so, please move the line after the comment, otherwise it probably would deserve a separate comment.